<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeachDlg</class>
 <widget class="QWidget" name="TeachDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>630</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>虚拟示教器</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/teachShow.png</normaloff>:/images/teachShow.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>文件名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fileName_label">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QListWidget" name="order_listWidget">
          <property name="maximumSize">
           <size>
            <width>38</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:rgba(240,240,240,255);color:#000000;border:0px solid gray;padding:0px;
item{width:94px;height:38px;border:0px solid gray;padding-left:24px;}
item:selected:active{background:#FFFFFF;color:#19649F;border-width:-1;}
item:selected{background:#FFFFFF;color:#19649F;}</string>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>空移</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>插补</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>圆弧</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>圆</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>开火</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>关火</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QToolButton" name="j1dec_button">
              <property name="text">
               <string>1-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="j1add_button">
              <property name="text">
               <string>1+</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>28</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="xdec_button">
              <property name="text">
               <string>X-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="xadd_button">
              <property name="text">
               <string>X+</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QToolButton" name="j2dec_button">
              <property name="text">
               <string>2-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="j2add_button">
              <property name="text">
               <string>2+</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>28</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="ydec_button">
              <property name="text">
               <string>Y-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="yadd_button">
              <property name="text">
               <string>Y+</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QToolButton" name="j3dec_button">
              <property name="text">
               <string>3-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="j3add_button">
              <property name="text">
               <string>3+</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_13">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="zdec_button">
              <property name="text">
               <string>Z-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="zadd_button">
              <property name="text">
               <string>Z+</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QToolButton" name="j4dec_button">
              <property name="text">
               <string>4-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="j4add_button">
              <property name="text">
               <string>4+</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>28</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="rdec_button">
              <property name="text">
               <string>R-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="radd_button">
              <property name="text">
               <string>R+</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QToolButton" name="j5dec_button">
              <property name="text">
               <string>5-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="j5add_button">
              <property name="text">
               <string>5+</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>28</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="pdec_button">
              <property name="text">
               <string>P-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="padd_button">
              <property name="text">
               <string>P+</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QToolButton" name="j6dec_button">
              <property name="text">
               <string>6-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="j6add_button">
              <property name="text">
               <string>6+</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>28</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="wdec_button">
              <property name="text">
               <string>W-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="wadd_button">
              <property name="text">
               <string>W+</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QRadioButton" name="point_move_radio">
                  <property name="text">
                   <string>点动</string>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="fixed_length_move">
                  <property name="text">
                   <string>定长</string>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QToolButton" name="toolButton">
                  <property name="text">
                   <string>停止</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>速度：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="speedEdit">
                  <property name="text">
                   <string>3000</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>度(mm)/min</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>距离：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="distanceEdit">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>度(mm)    </string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
     <zorder></zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
